spec/system/header_nav_spec.rb

```diff
@@ -80,6 +80,12 @@
                                                    page.first('.govuk-heading-xl').text
                                                  }.from('Your list').to('Manage users')
       end
+
+      it 'shows the "Performance tracking" link and can follow it' do
```

Comment:
Small point but I think it would make more sense to say 'shows the "Performance tracking" link and it can be followed'